transformer_engine/pytorch/float8_tensor.py

```diff
@@ -619,7 +597,7 @@ def __torch_dispatch__(cls, func, types, args, kwargs=None):
                     fp8_meta_index = dst._fp8_meta_index
                     scale = dst._fp8_meta[fp8_meta_key].scale[fp8_meta_index]
                     amax = dst._fp8_meta[fp8_meta_key].amax_history[0][fp8_meta_index]
-                    dst._scale_inv = scale.detach().view(1).reciprocal()
+                    dst._scale_inv.copy_(scale.detach().reciprocal())
```

Comment:
We can save a kernel launch:
```suggestion
                    torch.reciprocal(scale.detach(), out=dst._scale_inv)
```

Reply:
Gives the wrong output, let's debug in the future!